core/common/src/main/java/alluxio/underfs/options/CreateOptions.java

```diff
@@ -41,11 +44,19 @@ public static CreateOptions defaults() {
    * Constructs a default {@link CreateOptions}.
    */
   private CreateOptions() {
+    mCreateParent = true;
```

Comment:
is it possible to make `mCreateParent` default to false?
Seems like in most cases for LocalFS and HDFS, we explicitly ensure the caller sites to set this option to `false`. Would setting the default to false help identify any other places where this is set to true by accident?